apps/crn-server/src/data-providers/reminders.data-provider.ts

```diff
@@ -527,3 +694,48 @@ const getSortDate = (reminder: ReminderDataObject): DateTime => {
 
   return DateTime.fromISO(reminder.data.endDate);
 };
+
+const getAssociationNameAndType = (
+  researchOutput: ResearchOutputQueried,
+): {
+  associationType: 'team' | 'working group' | null;
+  associationName: string | null;
+} => {
+  if (
+    researchOutput.flatData &&
+    researchOutput.flatData.workingGroups &&
+    researchOutput.flatData.workingGroups.length > 0
+  ) {
+    return {
+      associationType: 'working group',
+      associationName:
+        researchOutput.flatData.workingGroups[0]?.flatData.title || null,
+    };
+  }
+  if (
+    researchOutput.flatData &&
+    researchOutput.flatData.teams &&
+    researchOutput.flatData.teams.length > 0
+  ) {
+    return {
+      associationType: 'team',
+      associationName:
+        researchOutput.flatData.teams[0]?.flatData.displayName || null,
+    };
+  }
```

Comment:
It could be a little simpler here

```suggestion
  if (
    researchOutput.flatData?.workingGroups &&
    researchOutput.flatData.workingGroups[0]
  ) {
    return {
      associationType: 'working group',
      associationName:
        researchOutput.flatData.workingGroups[0].flatData.title || null,
    };
  }
  if (researchOutput.flatData?.teams && researchOutput.flatData.teams[0]) {
    return {
      associationType: 'team',
      associationName:
        researchOutput.flatData.teams[0].flatData.displayName || null,
    };
  }
```